Dockerfile

```diff
@@ -47,7 +47,7 @@ RUN apt-get -qq update \
   && chmod +x /usr/local/bin/SymbolCollector.Console
 
 RUN curl -fsSL https://storage.googleapis.com/flutter_infra_release/releases/stable/linux/flutter_linux_3.7.1-stable.tar.xz -o /tmp/flutter_linux_3.7.1-stable.tar.xz \
-  && tar xf /tmp/flutter_linux_3.7.1-stable.tar.xz \
+  && tar xf /tmp/flutter_linux_3.7.1-stable.tar.xz -C /tmp \
```

Comment:
it would be more natural to put this in `/opt/flutter` instead of `/tmp`

Reply:
@asottile-sentry I'd rather do it in a separate PR and unblock myself asap, this is just missing for now, other things use `tmp` as well.
is this fine?